hanami.gemspec

```diff
@@ -31,7 +31,6 @@ Gem::Specification.new do |spec| # rubocop:disable Metrics/BlockLength
   spec.add_dependency "hanami-controller", "~> 2.0.alpha"
   spec.add_dependency "hanami-router",     "~> 2.0.alpha"
   spec.add_dependency "hanami-utils",      "~> 2.0.alpha"
-  spec.add_dependency "hanami-view",       "~> 2.0.alpha"
```

Comment:
@timriley 🤔 ???

Reply:
Yep, we don't need it as a hard dependency anymore (n.b. this line wasn't here either _before_ I added that first pass of view integration).

_At this point,_ I think the "Application view" code does sit better inside hanami-view, because it makes it a lot easier to maintain both the standalone view/application view behaviour in the same place (especially since I had to do all that module trickery to make it happen).

Reply:
It wasn't there because we didn't have the integration yet, as `hanami-view` 1.x wasn't compatible.

Removing the gem from here will make it optional, but we haven't planned yet for this scenario.

Reply:
We’ve discussed this scenario previously, and at that point when the view integration code was living inside hanami, we _had_ to make it a gemspec dependency.

That’s not a hard requirement at this point, so I figured we should only bring in hard dependencies when we will _really_ need them. Can we just leave it like this and see how the remaining aspects of the action/view/framework integration play out before making a decision?

Reply:
Ok, I’ll put it back for now. We can revisit once we see the full integration picture play out.

Reply:
@jodosha This is back in place now.